From db3d4f89457052be8681244054066ae6b91f1ef2 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Thu, 4 Jul 2024 18:54:57 +0200
Subject: [PATCH 1171/1695] dts: stm32: stm32mp13: fix GPIOs secure state

Define in stm32mp135f-dk DTS file that the GPIO controlling the
tamper test button (GPIO A6) is secure.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 579071448891a9322c99404f6eb37e615552afc9 dts: stm32: stm32mp13: fix GPIOs secure state]
Change-Id: I5bf86725d41ffb711fee1edf5c29c9e21c010109
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/391592
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 core/arch/arm/dts/stm32mp135f-dk.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp135f-dk.dts b/core/arch/arm/dts/stm32mp135f-dk.dts
index e85ca1cf4..f6e93de42 100644
--- a/core/arch/arm/dts/stm32mp135f-dk.dts
+++ b/core/arch/arm/dts/stm32mp135f-dk.dts
@@ -163,6 +163,10 @@
 	>;
 };
 
+&gpioa {
+	st,protreg = < (TZPROT(6)) >;
+};
+
 &gpiob {
 	st,protreg = < (TZPROT(9)) >;
 };
-- 
2.47.2

